translations/hu/get5.phrases.txt

```diff
@@ -2,346 +2,501 @@
 {
     "ReadyForMapSelectionInfoMessage"
     {
-        "hu"            "Használd a {1} parancsot, ha a csapatod készen áll a pálya választásra."
+        "hu"       "Használd a {1} parancsot, ha a csapatod készen áll a pálya választásra."
     }
     "WaitingForCastersReadyInfoMessage"
     {
-        "hu"            "A {1} várunk, hogy beírja a {2} parancsot a kezdéshez."
+        "hu"       "A {1} várunk, hogy beírja a {2} parancsot a kezdéshez."  
     }
     "ReadyToRestoreBackupInfoMessage"
     {
-        "hu"            "Használd a {1} parancsot, ha a készen állsz a mérkőzés visszatöltésre."
+        "hu"       "Használd a {1} parancsot, ha a készen állsz a mérkőzés visszatöltésre."
     }
     "ReadyToKnifeInfoMessage"
     {
-        "hu"            "Használd a {1} parancsot, ha készen állsz a késelésre."
+        "hu"       "Használd a {1} parancsot, ha készen állsz a késelésre." 
     }
     "ReadyToStartInfoMessage"
     {
-        "hu"            "Használd a {1} parancsot, ha készenállsz a kezdésre."
+        "hu"       "Használd a {1} parancsot, ha készenállsz a kezdésre."
     }
     "YouAreReady"
     {
-        "hu"            "Készen állsz a mérkőzés megkezdésére."
+        "hu"       "Készen állsz a mérkőzés megkezdésére."
     }
     "YouAreReadyAuto"
     {
-        "hu"            "INFO: Felkészült állapotba kerültél a játék aktívitásod által."
+        "hu"       "INFO: Felkészült állapotba kerültél a játék aktívitásod által."
     }
     "TypeUnreadyIfNotReady"
     {
-        "hu"            "Írd be, hogy {1} ha nem állsz még készen."
+        "hu"       "Írd be, hogy {1} ha nem állsz még készen."
```

Comment:
We changed to `Használd a` instead of `"Írd be`. Does it not apply here?

Reply:
Thanks for the mention. I did this at night and i changed it somehow.
btw. "Használd a"  meaning - "Use the", "Írd be" meaning - Type in something

Reply:
Another HU translator said we should use `Használd a` instead. I can't make this call, obviously, as I don't speak Hungarian.

Reply:
`Írd be` sounds unsuitable for a command description, but neither would `Használd a` be grammatically correct.
I would suggest using `Használd a {1} parancsot ha nem állsz még készen.`